openapiart/openapiartprotobuf.py

```diff
@@ -209,6 +209,23 @@ def _write_header(self, info_object):
         self._write('import "google/protobuf/descriptor.proto";')
         self._write('import "google/protobuf/empty.proto";')
 
+    def _get_integer_format(self, format, min, max):
```

Comment:
as has been source of my confusion above, we should prioritize checking format first. e.g. here's the step I would expect:
- If format is provided
  - If min-max not compatible with format, raise error
- If format is not provided
  - Based on min-max provided, return appropriate type
  - For this particular release, return int32 if compatible and add a TODO comment to remove it later

Reply:
done